r_hip_yaw_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: r_hip_yaw
    joint_speed: 0.2
    motor:
        id: 1
        init: 2047
        min: 0
        max: 4095

r_hip_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: r_hip_roll
    joint_speed: 0.2
    motor:
        id: 2
        init: 2047
        min: 0
        max: 4095
        
r_hip_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: r_hip_pitch
    joint_speed: 0.2
    motor:
        id: 3
        init: 2047
        min: 0
        max: 4095
                
r_knee_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: r_knee_pitch
    joint_speed: 0.2
    motor:
        id: 4
        init: 2047
        min: 0
        max: 4095

r_ankle_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: r_ankle_pitch
    joint_speed: 0.2
    motor:
        id: 5
        init: 2047
        min: 0
        max: 4095

r_ankle_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: r_ankle_roll
    joint_speed: 0.2
    motor:
        id: 6
        init: 2047
        min: 0
        max: 4095

l_hip_yaw_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: l_hip_yaw
    joint_speed: 0.2
    motor:
        id: 11
        init: 2047
        min: 0
        max: 4095

l_hip_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: l_hip_roll
    joint_speed: 0.2
    motor:
        id: 12
        init: 2047
        min: 0
        max: 4095

l_hip_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: l_hip_pitch
    joint_speed: 0.2
    motor:
        id: 13
        init: 2047
        min: 0
        max: 4095
                
l_knee_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: l_knee_pitch
    joint_speed: 0.2
    motor:
        id: 14
        init: 2047
        min: 0
        max: 4095

l_ankle_pitch_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: l_ankle_pitch
    joint_speed: 0.2
    motor:
        id: 15
        init: 2047
        min: 0
        max: 4095

l_ankle_roll_controller:
    controller:
        package: dynamixel_controllers
        module: joint_position_controller
        type: JointPositionController
    joint_name: l_ankle_roll
    joint_speed: 0.2
    motor:
        id: 16
        init: 2047
        min: 0
        max: 4095
